src/signals/incident/components/form/DescriptionInfo/DescriptionInfo.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import styled from 'styled-components';
+import { makeSelectSubCategories } from 'models/categories/selectors';
+import { breakpoint } from '@datapunt/asc-ui';
+import { fetchCategories } from 'models/categories/actions';
+import { makeSelectIncidentContainer } from 'signals/incident/containers/IncidentContainer/selectors';
+import { isAuthenticated } from 'shared/services/auth/auth';
+
+const DescriptionInfoStyle = styled.div`
```

Comment:
The variable name implies it only contains styling, which is not the case; it is a styled component. For naming consistency, I'd suggest using `DescriptionInfoWrapper` or something similar.